tox.ini

```diff
@@ -18,6 +18,13 @@ skip_install = True
 deps = flake8
 commands = flake8 src/twister2 tests
 
+[testenv:docs]
+deps = -rrequirements-docs.txt
+commands_pre = python -c "import shutil; shutil.rmtree('build/html', ignore_errors=True)"
+commands =
+	sphinx-apidoc -o docs/.apidoc src/twister2 -l -F
+	sphinx-build -a -E -j auto -b html -n docs build/html -v -T
```

Comment:
add docs to CI, you could even host latest version, see e.g. https://github.com/teslabs/spinner/blob/main/.github/workflows/docs.yml

Reply:
The is out plan for the future when twister V2 is ready to release.